project-fortis-pipeline/docs/background.md

```diff
@@ -55,3 +55,13 @@ development of Fortis, including:
 - [Spark Streaming connector for Bing](https://github.com/CatalystCode/streaming-bing)
 - [Spark Streaming connector for RSS](https://github.com/CatalystCode/streaming-rss-html)
 - [Advanced filtering options for Twitter Spark Streaming connector](https://github.com/apache/bahir/pull/43)
+
+## Known Docker Issues
```

Comment:
This low-level development information should probably go to [development-faq.md](https://github.com/CatalystCode/project-fortis/blob/126e5f658c466dc63893ec5faf0b1ce95d9dcf80/project-fortis-pipeline/docs/development-faq.md) (as a matter of fact, the Docker one is already there)